trento/xml/sso-integration.xml

```diff
@@ -0,0 +1,762 @@
+-e SAML_METADATA_URL=&lt;&lt;SAML_METADATA_URL&gt;&gt; \
+-e SAML_METADATA_CONTENT=&lt;&lt;SAML_METADATA_CONTENT&gt;&gt; \
+
+# Optional:
+-e SAML_IDP_NAMEID_FORMAT=&lt;&lt;SAML_IDP_NAMEID_FORMAT&gt;&gt; \
+-e SAML_SP_DIR=&lt;&lt;SAML_SP_DIR&gt;&gt; \
+-e SAML_SP_ENTITY_ID=&lt;&lt;SAML_SP_ENTITY_ID&gt;&gt; \
+-e SAML_SP_CONTACT_NAME=&lt;&lt;SAML_SP_CONTACT_NAME&gt;&gt; \
+-e SAML_SP_CONTACT_EMAIL=&lt;&lt;SAML_SP_CONTACT_EMAIL&gt;&gt; \
+-e SAML_SP_ORG_NAME=&lt;&lt;SAML_SP_ORG_NAME&gt;&gt; \
+-e SAML_SP_ORG_DISPLAYNAME=&lt;&lt;SAML_SP_ORG_DISPLAYNAME&gt;&gt; \
+-e SAML_SP_ORG_URL=&lt;&lt;SAML_SP_ORG_URL&gt;&gt; \
+-e SAML_USERNAME_ATTR_NAME=&lt;&lt;SAML_USERNAME_ATTR_NAME&gt;&gt; \
+-e SAML_EMAIL_ATTR_NAME=&lt;&lt;SAML_EMAIL_ATTR_NAME&gt;&gt; \
+-e SAML_FIRSTNAME_ATTR_NAME=&lt;&lt;SAML_FIRSTNAME_ATTR_NAME&gt;&gt; \
+-e SAML_LASTNAME_ATTR_NAME=&lt;&lt;SAML_LASTNAME_ATTR_NAME&gt;&gt; \
+-e SAML_SIGN_REQUESTS=&lt;&lt;SAML_SIGN_REQUESTS&gt;&gt; \
+-e SAML_SIGN_METADATA=&lt;&lt;SAML_SIGN_METADATA&gt;&gt; \
+-e SAML_SIGNED_ASSERTION=&lt;&lt;SAML_SIGNED_ASSERTION&gt;&gt; \
+-e SAML_SIGNED_ENVELOPES=&lt;&lt;SAML_SIGNED_ENVELOPES&gt;&gt; \
+
+...[other settings]...
+</programlisting>
+      </section>
+    </section>
+    <section xml:id="available-variables-2">
+      <title>Available variables</title>
+      <variablelist>
+        <varlistentry>
+          <term>
+            SAML_IDP_ID
+          </term>
+          <listitem>
+            <para>
+              SAML IDP id
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            SAML_SP_ID
+          </term>
+          <listitem>
+            <para>
+              SAML SP id
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            SAML_METADATA_URL
+          </term>
+          <listitem>
+            <para>
+              URL to retrieve the SAML metadata xml file. One of
+              <literal>SAML_METADATA_URL</literal> or
+              <literal>SAML_METADATA_CONTENT</literal> is required
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            SAML_METADATA_CONTENT
+          </term>
+          <listitem>
+            <para>
+              One line string containing the SAML metadata xml file
+              content (<literal>SAML_METADATA_URL</literal> has
+              precedence over this)
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            SAML_IDP_NAMEID_FORMAT
+          </term>
+          <listitem>
+            <para>
+              SAML IDP name id format, used to interpret the attribute
+              name. Whole urn string must be used (default value:
+              <literal>SAML IDP name id format, used to interpret the attribute name. Whole urn string must be used</literal>)
```

Comment:
This will become a render problem as `<literal>` does not add linebreaks. Use `<screen>` instead if you really want to have this whole long string.

Reply:
This is actually a copy/paste error. It will be changed by other value